src/config/prediction_settings.py

```
@@ -2,20 +2,21 @@
 class PredictionSettings:
     """予測設定を管理するクラス（年単位固定）"""
-    
+
     def __init__(self, start_year: int, forecast_end_t: int):
         """
         初期化
-        
+
         Args:
             start_year: 実績データの開始年
             forecast_end_t: 予測の最終期間（年単位）
```

Ruff (0.12.2)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)